### PR TITLE
create collectionId field

### PR DESCRIPTION
**Pull request by Pantani**
create `CollectionId` field to normalize collections API

### CODE REVIEW

File: collectibles.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package blockatlas
 
 type Collection struct {
+	CollectionId    string `json:"collection_id"`
```
Comment (vikmeup) [inline collectibles.go]: can we just call id `id` ? it's already returning collection model

Comment (madcake) [reply collectibles.go]: In client used collection_id. In next api version good idea use id.

Comment (vikmeup) [reply collectibles.go]: which client? `v2/ethereum/collections/0x5574Cd97432cEd0D7Caf58ac3c4fEDB2061C98fB` does not have `collection_id `

Comment (madcake) [reply collectibles.go]: I looked /collectibles/category.

Comment (vikmeup) [reply collectibles.go]: `id` should be good to go then!

**Pull request merged**